modules/trusted-firmware-m/Kconfig

```diff
@@ -11,6 +11,7 @@ config TFM_BOARD
 	string
 	default "nordic_nrf/nrf9160dk_nrf9160" if BOARD_NRF9160DK_NRF9160NS
 	default "nordic_nrf/nrf5340dk_nrf5340_cpuapp" if BOARD_NRF5340DK_NRF5340_CPUAPPNS
+	default "nordic_nrf/thingy53_nrf5340_cpuapp" if BOARD_THINGY53_NRF5340_CPUAPP
```

Comment:
This won't work as it is referencing platform folders inside the TFM repo. This should work automatically in sdk-nrf, though because of this:

https://github.com/nrfconnect/sdk-nrf/blob/master/modules/tfm/Kconfig#L10

Reply:
What I mean is, you should remove the tfm commit. You will get tfm support automatically in ncs without having to add the board anywhere.